open-company/open-company-web
# NUX Part XII

Card: https://trello.com/c/QFMgqhGl

To test 1 (Slack):
- signup with a new Slack team
- [x] do you land to the org setup? Good (no profile setup)
- proceed
- [x] did you go directly to the org dashboard, no invite users via email? Good
- [x] do you see the add post yellow tooltip when you get to the dashboard? Good
- dismiss it by clicking the X
- [x] do you NOT see the add bot notification appear? Good
- add a post
- [x] do you see the add bot notification appear? Good
- refresh
- [x] add bot notification is still there? Good
- dismiss it
- refresh
- [x] do you NOT see the add bot notification again? Good

Test 2 (Email):
- signup with email
- add profile info
- setup org
- do NOT invite other users
- [x] do you see the add post tooltip? Good
- enter edit mod
- add a post and publish it
- [x] do you NOT see the add post tooltip anymore? Good
- [x] do you see the invite people tooltip instead? Good
- delete the previously added post
- refresh
- [ ] invite people tooltip is still there? Good
- invite user A as viewer
- accept invitation as user A
- setup profile and proceed to dashboard
- [x] do you see the orange welcome overlay when you land on dashboard (you should see the add post but you are a viewer)
- [x] do you NOT see the add post or invite people tooltip when you dismiss the above overlay? Good
- as the admin user invite user B as contributor
- accept invitation as user B
- setup profile and proceed to dashboard
- [x] do you see the add post tooltip? Good
- add a post
- [x] did the add post tooltip goes away? Good
- [x] no other tooltip or notification shown after a refresh? Good
- as the initial admin user invite user C as admin
- accept invitation as user C
- setup profile and proceed to dashboard
- [x] do you see the orange welcome overlay? Good
- dismiss it
- [x] do you NOT see the add post tooltip? Good
- [x] do you NOT see the invite people tooltip? Good (there are already invited users)
